docs/starters.yml

```
@@ -5161,3 +5161,15 @@
     - React Scroll for one page, anchor based navigation is available
     - Code highlighting
     - SEO optimized to include social media images and twitter handles
+- url: https://gatsby-starter-default-dark-mode.netlify.com/
+  repo: https://github.com/alexandreramosdev/gatsby-starter-default-dark-mode
+  description: A simple starter to get up and developing quickly with Gatsby, dark mode a styled-components.
```

Typo here :) Maybe this?
```suggestion
  description: A simple starter to get developing quickly with Gatsby, dark mode, and styled-components.
```

---

Thanks!  My english not is better yet